src/khoj/interface/web/chat.html

```diff
@@ -563,6 +564,148 @@
             document.getElementById("chat-body").scrollTop = document.getElementById("chat-body").scrollHeight;
         }
 
+        function openFileBrowser() {
+            event.preventDefault();
+            var overlayText = document.getElementById("dropzone-overlay");
+            var dropzone = document.getElementById('chat-body');
+
+            if (overlayText == null) {
+                dropzone.classList.add('dragover');
+                var overlayText = document.createElement("div");
+                overlayText.innerHTML = "Select a file to share it with Khoj";
+                overlayText.className = "dropzone-overlay";
+                overlayText.id = "dropzone-overlay";
+                dropzone.appendChild(overlayText);
+            }
+
+            const fileInput = document.createElement('input');
+            fileInput.type = 'file';
+            fileInput.addEventListener('change', function() {
+                const selectedFile = fileInput.files[0];
```

Comment:
Does this file attachment button only support uploading 1 file at a time?

Reply:
Yes, that's the default button of the component. It can be updated to take multiple as well. We can pick that up later if there's demand.